apps/antalmanac/src/components/RightPane/SectionTable/SectionTableBody.tsx

```diff
@@ -19,6 +22,16 @@ import { mobileContext } from '$components/MobileHome';
 import locationIds from '$lib/location_ids';
 import { translateWebSOCTimeTo24HourTime } from '$stores/calendarizeHelpers';
 
+const DAYS_TO_NUMS: { [key: string]: number } = {
```

Comment:
This is moved outside the function component because it's a const value and we don't really need it re-calculated on every render.